src/status_im/utils/universal_links/core.cljs

```diff
@@ -76,7 +77,24 @@
 (rf/defn handle-community
   [cofx {:keys [community-id]}]
   (log/info "universal-links: handling community" community-id)
-  (navigation/navigate-to-cofx cofx :community {:community-id community-id}))
+  (navigation/navigate-to-cofx cofx :community {:community-id community-id})
+)
+
+(rf/defn handle-navigation-to-desktop-community-from-mobile
+  {:events [:handle-navigation-to-desktop-community-from-mobile]}
+  [{:keys [db]} cofx de-serialised-key]
+  (navigation/navigate-to-cofx cofx :community {:community-id de-serialised-key})
+)
+
+(rf/defn handle-desktop-community
+  [cofx {:keys [community-id]}]
+  (status/deserialize-and-compress-key
+   community-id
+   (fn [de-serialised-key]
```

Comment:
I think we should try to be as consistent as possible with naming. This PR is using the word `deserialize` in multiple ways, sometimes using the British convention with `s`, sometimes with a dash. I suggest we stick to only one way. The dash seems unnecessary though.

Reply:
Okay I will make it `deserialize` since that is how its present in status-go. more consistency :)